fluid/SE-ResNeXt-152/train.py

```diff
@@ -320,10 +324,23 @@ def train_parallel_exe(args):
     class_dim = 1000
     image_shape = [3, 224, 224]
 
-    main = fluid.Program()
-    startup = fluid.Program()
-
-    with fluid.program_guard(main, startup):
+    #main = fluid.Program()
+    #startup = fluid.Program()
+
+    #with fluid.program_guard(main, startup):
+    if args.with_feeder:
```

Comment:
It seems the `with_feeder` is not added to argument parser.

Reply:
Done, thanks!